build.gradle

```diff
@@ -1,3 +1,8 @@
+plugins {
+  id "org.owasp.dependencycheck" version "1.4.5"
+  id "com.github.johnrengelman.shadow" version "1.2.4"
```

Comment:
So, I did some digging. Seems like gradle plugins block uses http://plugins.gradle.com/m2 which simply redirects all requests to jcenter.

```
~ curl -I https://plugins.gradle.org/m2/junit/junit/4.12/junit-4.12.pom
HTTP/1.1 303 See Other
Date: Fri, 27 Jan 2017 10:32:44 GMT
Content-Length: 0
Connection: keep-alive
Set-Cookie: __cfduid=dfb3ae2221aa16499509023673d8240ee1485513164; expires=Sat, 27-Jan-18 10:32:44 GMT; path=/; domain=.gradle.org; HttpOnly
Location: https://jcenter.bintray.com/junit/junit/4.12/junit-4.12.pom
Via: 1.1 vegur
Server: cloudflare-nginx
CF-RAY: 327b8adfa92f0a00-ATL
```

Also, it seems that the gradle plugins website keeps the classpath mapping, as documented in their submission and reclaiming of groupIds docs.
https://plugins.gradle.org/docs/submit
https://plugins.gradle.org/docs/reclaiming